### PR TITLE
DEP PHP Support in CMS5

### PR DESCRIPTION
**Pull request by sabina-talipova**
## Parent issue
- https://github.com/silverstripe/silverstripe-framework/issues/10402

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -5,45 +5,48 @@
...
+        "silverstripe/gridfieldqueuedexport": "3.x-dev",
+        "silverstripe/ldap": "2.x-dev",
+
+        "silverstripe/textextraction": "4.x-dev",
+        "silverstripe/realme": "5.x-dev",
+        "silverstripe/ckan-registry": "2.x-dev",
+        "silverstripe/webauthn-authenticator": "5.x-dev",
+        "silverstripe/security-extensions": "5.x-dev",
+        "silverstripe/subsites": "3.x-dev",
+        "silverstripe/html5": "3.x-dev",
```
Comment (maxime-rainville) [inline composer.json]:
```suggestion
```

We're not going to need this module going forward.

https://github.com/silverstripe/silverstripe-framework/issues/10528

Comment (sabina-talipova) [reply composer.json]: DONE

**Comment by emteknetnz** [comment]
Requires https://github.com/silverstripe/silverstripe-frameworktest/pull/135 to be merged first

**Comment by emteknetnz** [comment]
We'll deal with unit test failures in various modules in a follow broken builds card.  For now the main this was just get recipe-kitchen-sink installable

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -25,10 +25,11 @@ jobs:
...
-          phpunit: true
-          phpunit_suite: recipe-cms
         - php: 8.1
           phpunit: true
           phpunit_suite: recipe-cms
+      # TODO: renable prefer-lowest build once CMS 5 beta is out
+      # - php: 8.1
+      #   composer_args: --prefer-lowest
+      #   phpunit: true
+      #   phpunit_suite: recipe-cms
```
Comment (ChloeSartorelli) [inline .github/workflows/ci.yml:35]: I've added this to https://github.com/silverstripe/gha-generate-matrix/issues/41

File: composer.json [hunk 3]
```diff
@@ -5,45 +5,43 @@
...
-        "cwp/starter-theme": "3.x-dev",
-        "cwp/agency-extensions": "2.x-dev",
-        "cwp/watea-theme": "3.x-dev",
-        "cwp/cwp": "2.x-dev",
-        "cwp/cwp-pdfexport": "1.x-dev",
-        "dnadesign/silverstripe-elemental-userforms": "~3.2.1@stable",
-        "symbiote/silverstripe-multivaluefield": "5.x-dev",
-        "tractorcow/silverstripe-fluent": "4.x-dev"
+        "php": "^8.1",
+        "silverstripe/recipe-plugin": "^2",
```
Comment (ChloeSartorelli) [inline composer.json]: Won't this need to be the x-dev constraint as per https://github.com/silverstripe/silverstripe-installer/pull/340? Or will the fact that it's x-dev in installer take precedence already?

Comment (emteknetnz) [reply composer.json]: Ah yup, have changed to 2.x-dev and put a note on https://github.com/silverstripe/silverstripe-installer/issues/341 to also handle recipe-kitchen-sink

**Review by GuySartorelli — APPROVED** [review]
Merge on (relatively) green

**Pull request merged**